.rubocop.yml

```diff
@@ -29,7 +29,7 @@ Metrics/AbcSize:
   Enabled: false
 
 Metrics/ClassLength:
-  Max: 200
+  Max: 210
```

Comment:
200 is the maximum recommended length, keep it that way

Reply:
I will put the resource_provider.rb into exclude clause, ok?!
Because the resource_provider.rb has more than 200 lines. Do you know some option better instead of put resource_provider.rb into exclude clause?

Reply:
2 Things:
1- Make methods more generic instead of adding many methods to do really similar stuff, this is ruby 101.
2- Worst comes to worst, not `everything` has to be inside resource_provider, you could just create a new file and include that, right?

Reply:
Nevermind, it is related to the class length, it is fine